### PR TITLE
Fixing error with loading of monitoring data and plots

### PR DESCRIPTION
**Pull request by rupozzi**
The monitoring in the certification web app was not loading, both the data and the plots due to an error in nomenclature in MonitoringHandler.

BEGINRELEASENOTES

FIX: Fixing nomenclature of arguments for getSelectionData and getPlotImg that 

ENDRELEASENOTES

### CODE REVIEW

**Comment by rupozzi** [comment]
Wait to merge until #642 is merged

File: src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py
Comment (fstagni) [inline src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py]: why not "fileName"?

Comment (rupozzi) [reply src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py]: I believe the name was the root of the error

**Review by fstagni — APPROVED** [review]

**Comment by fstagni** [comment]
But please fix what `pylint` reports

**Comment by rupozzi** [comment]
@fstagni the pylint error comes from the fact that we need to wait until #642 is merged, since there we removed the monitoringClient from the FS

**Comment by fstagni** [comment]
There's a conflict now.

**Comment by rupozzi** [comment]
> There's a conflict now.

Resolved now

**Pull request merged**
